### DISCUSSION TITLE
Can it be used for commercial purposes within the company?

### DISCUSSION THREAD

**Discussion by Lisnabin** [WinMerge/winmerge#2563] [Q&A]
There are no plans to change or modify the source.
I'm only going to use it, can I use it for commercial purposes within the company?

**Comment by sdottaka** [comment]
Yes, you can use WinMerge for commercial purposes within your company. The GPLv2 license allows unrestricted use, including in commercial environments, as long as you don’t distribute modified versions outside your company without providing the source code.
